lib/services/audio_service_helper.dart

```diff
@@ -64,17 +64,44 @@ class AudioServiceHelper {
     }
   }
 
+  bool hasQueueItems() {
+    return (_audioHandler.queue.valueOrNull?.length ?? 0) != 0;
+  }
+
+  @Deprecated("Use addQueueItems instead")
   Future<void> addQueueItem(BaseItemDto item) async {
+    await addQueueItems([item]);
+  }
```

Comment:
Should this be marked deprecated? It would be more ergonomic to allow adding single items, and would remove the need to change everything that adds a single item